pandas/core/generic.py

```diff
@@ -8839,6 +8839,12 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
             extend the index when shifting and preserve the original data.
         axis : {0 or 'index', 1 or 'columns', None}, default None
             Shift direction.
+        fill_value : object, optional
+            the scalar value to use for newly introduced missing values.
+            the default depends on the dtype of `self`. for numeric data,
+            ``np.nan`` is used. for datelike, ``pandas.nat`` is used.
```

Comment:
Capitalize.

Maybe replace dateline with "datetime, timedelta, or period data, ..."

change \`\`pandas.nat\`\` to :attr:\`NaT\`.

Reply:
And could you add that "For extension dtypes, ``self.dtype.na_value`` is used".

Reply:
done